### PR TITLE
chore: Remove the end-to-end-encryption keyword

### PR DESCRIPTION
**Pull request by poljar**
Apparently keywords need to have less than 20 characters.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ description = "A Rust implementation of Olm and Megolm"
 repository = "https://github.com/matrix-org/vodozemac"
 version = "0.6.0"
 edition = "2021"
-keywords = ["matrix", "chat", "messaging", "olm", "end-to-end-encryption"]
+keywords = ["matrix", "chat", "messaging", "olm"]
```
Comment (dkasak) [inline Cargo.toml:11]:
```suggestion
keywords = ["matrix", "chat", "messaging", "olm", "megolm", "double ratchet", "encryption", "E2EE", "cryptography"]
```

Comment (dkasak) [reply Cargo.toml:11]: Should we add a bunch more relevant ones that are < 20 chars while we're at it?

**Review by dkasak — APPROVED** [review]
LGTM, but see suggestion.

File: Cargo.toml (hunk 1 above)
Comment (poljar) [reply Cargo.toml]: You can have at most 5 keywords per crate. :face_exhaling:

**Pull request merged**